LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2016, Starweb AB
+Copyright (c) 2016-2018, Starweb AB
```

Comment:
A copyright notice with only the year of first publication is easier to manage (https://en.wikipedia.org/wiki/Copyright_notice).